meta-resin-common/recipes-containers/docker/docker/docker.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
```

Comment:
This file entirely overwrites the changes you do to the docker.service in a previous commit.

Please redo the commits in the correct order